htdocs/xoops_lib/modules/protector/library/HTMLPurifier/AttrTransform/TargetNoreferrer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+// must be called POST validation
+
+/**
+ * Adds rel="noreferrer" to any links which target a different window
+ * than the current one.  This is used to prevent malicious websites
+ * from silently replacing the original window, which could be used
+ * to do phishing.
+ * This transform is controlled by %HTML.TargetNoreferrer.
+ */
+class HTMLPurifier_AttrTransform_TargetNoreferrer extends HTMLPurifier_AttrTransform
```

Comment:
__[phpmd]__
* The class HTMLPurifier_AttrTransform_TargetNoreferrer is not named in CamelCase.

<!-- Comment by [SideCI](https://sideci.com) -->